tests/e2e/test_slippage_investigation.py

```diff
@@ -14,7 +14,7 @@ def test_no_outrageous_slippage(self):
         which tx are having high slippage values in dollar terms
         """
         dune = DuneAnalytics.new_from_environment()
-        period_start = datetime.strptime('2022-03-10', "%Y-%m-%d")
+        period_start = datetime.strptime('2022-03-01', "%Y-%m-%d")
```

Comment:
By changing these dates to be 10 days, this test has become more of a "Benchmarking" test than anything else. Do we know how long it takes to execute? Seems like a waste of CI and should probably be ignored.